pkg/translate/translate.go

```diff
@@ -234,60 +236,70 @@ var (
 					AlwaysEnabled:        true,
 				},
 				name.PilotComponentName: {
+					ResourceType:         "Deployment",
```

Comment:
change to string constant (would be better to reference k8s def but it seems like that's not available)

Reply:
done